### PR TITLE
Bring back old features

### PR DESCRIPTION
**Pull request by earthchie**
- Improve some print output

# Bring back from previous master
- Show number of balls on bot init
- Ball type decision maker (CP > 300 Great Ball, CP > 700 Ultra Ball)

### CODE REVIEW

**Comment by earthchie** [comment]
image

File: pokemongo_bot/cell_workers/seen_fort_worker.py [hunk 1]
```diff
@@ -26,9 +26,9 @@ def work(self):
         fortID = self.fort['id']
         dist = distance(self.position[0], self.position[1], lat, lng)
 
-        print('[#] Found fort {} at distance {}m'.format(fortID, dist))
+        print('Found fort {} at distance {}m'.format(fortID, dist))
         if dist > 10:
```
Comment (douglascamata) [inline pokemongo_bot/cell_workers/seen_fort_worker.py]: Please keep the log format in this file, with `[#]` and `[+]`

File: pokemongo_bot/__init__.py [hunk 2]
```diff
@@ -34,8 +34,6 @@ def work_on_cell(self, cell, position, include_fort_on_path):
         if (self.config.mode == "all" or self.config.mode == "poke") and 'catchable_pokemons' in cell:
             print '[#] Something rustles nearby!'
             for pokemon in cell['catchable_pokemons']:
-                with open('web/catchable.json', 'w') as outfile:
-                    json.dump(pokemon, outfile)
```
Comment (douglascamata) [inline pokemongo_bot/__init__.py]: Why was this removed?

File: pokemongo_bot/__init__.py [hunk 3]
```diff
@@ -91,8 +89,6 @@ def _setup_api(self):
         inventory_req = self.api.call()
 
         inventory_dict = inventory_req['responses']['GET_INVENTORY']['inventory_delta']['inventory_items']
-        with open('web/info.json', 'w') as outfile:
-            json.dump(inventory_dict, outfile)
```
Comment (douglascamata) [inline pokemongo_bot/__init__.py]: Why was this removed?

**Comment by douglascamata** [comment]
@earthchie your branch has conflicts with `dev`, please update from the original branch.

File: pokemongo_bot/__init__.py (hunk 2 above)
Comment (earthchie) [inline pokemongo_bot/__init__.py]: just conflict

File: pokemongo_bot/__init__.py (hunk 3 above)
Comment (earthchie) [inline pokemongo_bot/__init__.py]: same above. conflict.

File: pokemongo_bot/cell_workers/seen_fort_worker.py (hunk 1 above)
Comment (earthchie) [inline pokemongo_bot/cell_workers/seen_fort_worker.py]: ok

**Comment by earthchie** [comment]
@douglascamata conflicts solved :)

**Comment by earthchie** [comment]
@douglascamata oh wait. got some errors

image

**Comment by douglascamata** [comment]
@earthchie do you mind getting the latest `dev` code and add your modifications again (only "new" stuff)? I can clearly see the features you added, but I think somehow the merge request got messed up and there are a lot of things that shouldn't be there.

**Comment by earthchie** [comment]
I did, I pulled from upstream everytime before I make a new commit.

but it still messed up. dunno why...

**Comment by earthchie** [comment]
@douglascamata solved all conflicts again. tested. working now.

File: pokemongo_bot/__init__.py [hunk 4]
```diff
@@ -34,7 +34,7 @@ def work_on_cell(self, cell, position, include_fort_on_path):
         if (self.config.mode == "all" or self.config.mode == "poke") and 'catchable_pokemons' in cell:
             print '[#] Something rustles nearby!'
             for pokemon in cell['catchable_pokemons']:
-                with open('web/catchable.json', 'w') as outfile:
+                with open('web/info.json', 'w') as outfile:
```
Comment (douglascamata) [inline pokemongo_bot/__init__.py]: @earthchie I'm afraid you're going to break map functionality here. Remove this change from the merge request?

**Comment by douglascamata** [comment]
@earthchie just one last thing to merge!

File: pokemongo_bot/__init__.py (hunk 4 above)
Comment (earthchie) [inline pokemongo_bot/__init__.py]: @douglascamata done!

**Comment by douglascamata** [comment]
@earthchie thank you very much and sorry for  being annoying, but that's the merge police duty. 👮

**Comment by earthchie** [comment]
thanks. that's okay :)
